### PR TITLE
move contributing section into the getting started section

### PR DESCRIPTION
**Pull request by luisaFelixSalles**
Doc changes:

- moves contributing section into the getting started section
- navigation bar don't have "more" anymore

image

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -91,7 +91,8 @@
 
 redirects = {
      "user_guide/getting_started_with_dpf_server": "../getting_started/dpf_server.html",
-     "concepts/index": "../user_guide/index.html#concepts"
+     "concepts/index": "../user_guide/index.html#concepts",
+     "contributing": "source/getting_started/contributing.html"
```
Comment (PProfizi) [inline doc/source/conf.py:95]:
```suggestion
     "contributing": "getting_started/contributing.html"
```
Isn't that the correct new path?

File: doc/source/getting_started/index.rst [hunk 2]
```diff
@@ -111,11 +111,14 @@ To use PyDPF-Core, in the same Python environment, run this command:
 
 
 .. toctree::
+   :maxdepth: 2
    :hidden:
+   :caption: Introduction
```
Comment (PProfizi) [inline doc/source/getting_started/index.rst:116]: @luisaFelixSalles what does this render like? I think that no section title would be better for now. Until we have at least two sections.

Comment (PProfizi) [inline doc/source/getting_started/index.rst:116]:
```suggestion
```

Comment (luisaFelixSalles) [reply doc/source/getting_started/index.rst]: image

**Pull request merged**